Resources/ServerInfo/Guidebook/Engineering/PipeNetworks.xml

```diff
@@ -0,0 +1,49 @@
+﻿<Document>
+  # Pipe Networks
+  Pipe networks — commonly called Pipenets — are a series of interconnected pipes, usually with a common task.
+
+  <Box>
+    <GuideEntityEmbed Entity="GasPipeStraight" Caption=""/>
+    <GuideEntityEmbed Entity="GasPipeHalf" Caption=""/>
+    <GuideEntityEmbed Entity="GasPipeBend" Caption=""/>
+    <GuideEntityEmbed Entity="GasPipeTJunction" Caption=""/>
+    <GuideEntityEmbed Entity="GasPipeFourway" Caption=""/>
+    <GuideEntityEmbed Entity="GasPressurePump" Caption=""/>
+    <GuideEntityEmbed Entity="GasVolumePump" Caption=""/>
+  </Box>
+
+  Pipenets are used all throughout the station to deliver, remove, or otherwise move gas throughout it.
+  Great examples are the station's Distro or Wastenet, which deliver breathable air and remove waste gas, respectively.
+
+  ## How Pipenets Work
+  In Space Station 14, pipenets are not a whole, defined volume, with input and output points. Instead, each pipe has its own separate volume.
```

Comment:
I'm only responding to this one for now because I saw this at like 1am

They do? I could have sworn there was a PR that makes them behave like this. And this is behavior I have partially observed. Or does the gas analyzer really just let you see the volume of only one pipe?

Reply:
> I'm only responding to this one for now because I saw this at like 1am
> 
> They do? I could have sworn there was a PR that makes them behave like this. And this is behavior I have partially observed. Or does the gas analyzer really just let you see the volume of only one pipe?

the gas analyzer trolls you by just showing you what one pipe's contents would look like, it's doing maths to show you specifically that

Reply:
are you fuckciing kidding me man